conans/test/integration/command_v2/test_info_build_order.py

```diff
@@ -134,7 +134,7 @@ def test_info_build_order_options():
                         "context": "build",
                         "binary": "Build",
                         "options": [
-                                "tool:myopt=1"
+                                "tool/0.1#b4c19a1357b43877a2019dd2804336a9:myopt=1"
```

Comment:
It seems the revision is not necessary and ``--build=tool/0.1`` or ``-o tool/0.1:myopt=1`` will work perfectly too. Do we want to simplify here?